inst/templates/content-reference-index.html

```diff
@@ -12,21 +12,27 @@ <h1>Reference</h1>
       <col class="title" />
     </colgroup>
 
-    {{#sections}}<tbody>
-      <tr>
+    {{#rows}}<tbody>
+      {{#title}}<tr>
         <th colspan="{{#has_icons}}3{{/has_icons}}{{^has_icons}}2{{/has_icons}}">
           <h2 id="{{slug}}" class="hasAnchor"><a href="#{{slug}}" class="anchor"></a>{{title}}</h2>
           <p class="section-desc">{{{desc}}}</p>
         </th>
-      </tr>
-      {{#contents}}<tr>
+      </tr>{{/title}}
+      {{#subtitle}}<tr>
+        <th colspan="{{#has_icons}}3{{/has_icons}}{{^has_icons}}2{{/has_icons}}">
+          <h3 id="{{slug}}" class="hasAnchor"><a href="#{{slug}}" class="anchor"></a>{{subtitle}}</h3>
+          <p class="section-desc">{{{desc}}}</p>
```

Comment:
Delete this `section-desc` so that the `desc` only labels `titles` and not each `subtitle`.